app/js/script.js

```diff
@@ -0,0 +1,9 @@
+import { addBook, insertBook, removeLoad } from './utilities.js';
+
+const addBtn = document.querySelector('.book-add');
+
+addBtn.addEventListener('click', () => {
+  addBook();
+  insertBook();
+  removeLoad();
```

Comment:
**OPTIONAL** - I suggest you do not need to call this method here. Please remove this line of code 🙏 🙏